tests/src/test/scala/cats/tests/ExtraRegressionSuite.scala

```diff
@@ -1,25 +0,0 @@
-package cats.tests
-
-import cats.Show
-import ExtraRegressionSuite._
-
-class ExtraRegressionSuite extends CatsSuite {
-
-  /**
-   * Placed here to work around scala/bug#6260 on scala 2.10
-   */
-  test("#1802 duplicated method name") {
```

Comment:
Deleted this for now, since it was giving me an error and seems like this is pretty old